internal/factsengine/gatherers/packageversion.go

```diff
@@ -50,19 +54,85 @@ func (g *PackageVersionGatherer) Gather(factsRequests []entities.FactRequest) ([
 			continue
 		}
 
-		version, err := g.executor.Exec(
-			"rpm", "-q", "--qf", "%{VERSION}", factReq.Argument)
+		packageName := factReq.Argument
+		requestedVersion := ""
+		if strings.Contains(factReq.Argument, ",") {
+			arguments := strings.SplitN(factReq.Argument, ",", 2)
+			packageName = arguments[0]
+			requestedVersion = arguments[1]
+		}
+
+		version, err := executeRpmVersionRetrieveCommand(g.executor, packageName)
 		if err != nil {
-			gatheringError := PackageVersionCommandError.Wrap(factReq.Argument)
+			gatheringError := PackageVersionCommandError.Wrap(packageName)
```

Comment:
Do we need to wrap the error again? It is already wrapped and logged in `executeRpmVersionRetrieveCommand`.
It loos redundant